constraints/miden-vm/hash.air

```diff
@@ -34,27 +35,38 @@ fn get_f_mv(s: vector[3]) -> scalar:
 fn get_f_mu(s: vector[3]) -> scalar:
     return cycle_row_0 & s[0] & s[1] & s[2]
 
+# Set to 1 when selector flags are (0,0,0) on rows which are 1 less than a multiple of 8. This is 
+# flag of the instruction that returns the result of the currently running computation.
+fn get_f_hout(s: vector[3]) -> scalar:
+    return cycle_row_7 & !s[0] & !s[1] & !s[2]
+
+
+# Set to 1 when selector flags are (0,0,1) on rows which are 1 less than a multiple of 8. This is 
+# flag of the instruction that returns the whole hasher state.
```

Comment:
nit: I would change the second sentence to "This flag indicates that the entire hasher state is returned."